app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -126,6 +126,15 @@
       @extend .crayons-btn;
     }
 
+    .back-button {
+      @extend .crayons-btn;
+      @extend .crayons-btn--ghost;
+      @extend .crayons-btn--icon;
+
+      align-items: center;
+      display: flex;
```

Comment:
This was added to make sure the icon lines up correctly with the other items in the `div.navigation-content`, which is a flex container in and of itself.